src/benchcab/internal.py

```diff
@@ -30,6 +30,10 @@
 
 # DIRECTORY PATHS/STRUCTURE:
 
+# Path to hidden state directory:
+STATE_DIR = Path(".state")
+STATE_PREFIX = ".attr_"
```

Comment:
What does the STATE_PREFIX give us? Is it in case we want to use the same functionality for something else and we need to differentiate the various files created?

Reply:
I prefixed the state files to differentiate them from ordinary files and are hidden so that users are less likely to accidentally break something. The `.attr_` prefix isn't that descriptive now that I'm thinking about it -  I will change this to `.state_attr_`.